classes/Domain/CallForProposal.php

```diff
@@ -2,7 +2,9 @@
 
 namespace OpenCFP\Domain;
 
-use DateTime;
+use DateTimeInterface;
```

Comment:
please reorder your `use` statements in alphabetical order to prevent triggering the Code Sniffer error `ordered_imports` in Travis.